### PR TITLE
single review

### PR DESCRIPTION
**Pull request by Cristian-pacheco**

### CODE REVIEW

**Review by GoJhon — COMMENTED** [review]
Hi Cristian, you have a nice logic, but you need to fix your flowchart; in the decision c<=3000000, you forgot put "yes" and "no" on the flows, fix height on the flow and fix some variable.

**Comment by hguzman** [comment]
@GoJhon excellent comment.
@Cristian-pacheco Please fix it.

**Review by LuisJrDev — APPROVED** [review]
Good job!

File: cristianpacheco/codigo.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+```
+Sub b()
+     m = 0
+     i = 0
+     While c <= 2999999
```
Comment (EmanuelObredor18) [inline cristianpacheco/codigo.md:5]: I have seen this in a limited number of codes. Good logic.

File: cristianpacheco/datos.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+| **Nombre completo** | cristian davith pacheco martinez |
+| **Edad** | 18 |
+| **Dirección** | kr 12#60c-10|
+| **Barrio** | nuevo milenio |
+| **Celular** | 3105027548|
+| **Email** | cristianpacheco10a@gmail.com |
+| **GitHub** |[cristian-pacheco](https://github.com/Cristian-pacheco) |
+
+## Ejercicios
+- [ciclo mientras que en codigo](cristianpacheco\codigo.md)
```
Comment (EmanuelObredor18) [inline cristianpacheco/datos.md]: The links are down. Please check your md

**Review by EmanuelObredor18 — APPROVED** [review]
Everything is fine in your code, not counting the broken links in the resume. I'll approve, but check your markdown file.

**Review by GoJhon — APPROVED** [review]
Hi @Cristian-pacheco, good job! I see your activity "buscador" and you have a good code. It's no necessary put a "sw" because you weren't use it, but it's alright. Keep it up!

**Review by GoJhon — CHANGES_REQUESTED** [review]
@Cristian-pacheco, please create a new branch for pull request. This pull request is into main from main.

**Review by Jesusvillanueva07 — APPROVED** [review]
good job. @Cristian-pacheco

**Review by juanhsd — APPROVED** [review]
Well done

**Pull request merged**